meinberlin/apps/projects/views.py

```diff
@@ -327,3 +328,8 @@ class DashboardProjectParticipantsView(AbstractProjectUserInviteListView):
     success_message_removal = _('Participant successfully removed.')
 
     invite_model = models.ParticipantInvite
+
+
+class DashboardProjectListView(a4dashboard_views.ProjectListView):
```

Comment:
i'd prefer to put this to the dashboard/ app as it overwrites functionality from the dashboard2